link-canonical/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "link-canonical"
+version = "0.1.0"
+authors = ["Kim Altintop <kim@eagain.st>"]
```

Comment:
Is there a reason we're not using "The Radicle Team" out of interest?

Reply:
Because it did not show up in `git log` :shrug:

Reply:
The cargo reference for [`[authors]`](https://doc.rust-lang.org/cargo/reference/manifest.html#the-authors-field) says: 

> The exact meaning is open to interpretation — it may list the original or primary authors, current maintainers, or owners of the package.

My personal interpretation is that maintainers are practically the same as package owners (on crates.io), and the `authors` field is never actually rendered anywhere (at least not that I'm aware of). We also use "The Radicle Contributors" in the license header, so as to not having to update it whenever some substantial modification occurs (although one would be free to do so). Additionally, "dev@radicle.xyz" is not actually a contact address.

So, "original or primary authors" seems sensible to me.

Reply:
Ya, sounds good to me too :+1: I was just curious :)